privacyidea/static/components/config/views/config.token.daypassword.html

```diff
@@ -0,0 +1,37 @@
+<h2 translate>DayPassword Token settings</h2>
+<p class="help-block" translate>
+    The DayPassword Token is a time based password token.
+    It is based on the totp token, but you can use it more than ones.</a>.
+</p>
+<p translate>
+    Here you can define settings, that will be set as default values, when
+    enrolling a DayPassword token.
+</p>
+
+<div class="form-group">
+    <label for="daypasswordStep" translate>Default Time Step</label>
+    <input type="text"
+           required
+           class="form-control"
+           placeholder="60"
+           ng-model="form['daypassword.timeStep']" name="daypasswordStep">
+</div>
+
+
+<div class="form-group">
```

Comment:
didn't we say we wanted to remove the timeshift?

Reply:
I removed only the auto-correction. We can still have a time shift by set up and we can also resync

Reply:
https://github.com/privacyidea/privacyidea/pull/3602/commits/1d33e4a5414b7c3962159450ba40a6e9f9e29488